From db7b84ca356024945c45f90dca7b1965de8dc513 Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Wed, 5 Apr 2017 10:40:34 +0530
Subject: [PATCH 055/532] spi: zynqmp: Cut the clocks after restore

commit 38c89870deda75b56621a0ff5b396d24ef1519a5 from
https://github.com/Xilinx/linux-xlnx.git

In the resume path the the clocks are not cut after restore.
Fix the same.

Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/spi/spi-zynqmp-gqspi.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/spi/spi-zynqmp-gqspi.c b/drivers/spi/spi-zynqmp-gqspi.c
index fce3bd3..96a1649 100644
--- a/drivers/spi/spi-zynqmp-gqspi.c
+++ b/drivers/spi/spi-zynqmp-gqspi.c
@@ -1059,6 +1059,9 @@ static int __maybe_unused zynqmp_qspi_resume(struct device *dev)
 	zynqmp_qspi_init_hw(xqspi);
 	spi_master_resume(master);
 
+	clk_disable(xqspi->refclk);
+	clk_disable(xqspi->pclk);
+
 	return 0;
 }
 
-- 
2.7.4

